io-package.json

```diff
@@ -1,8 +1,20 @@
 {
   "common": {
     "name": "iot",
-    "version": "1.11.4",
+    "version": "1.11.5",
```

Comment:
Please do not increase version and do not add news entries (especially not empty ones!!) ... this is done by release script based on redme changelog entries

Reply:
SO please add a new README changelog entry